package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "sdc-ia-scripts",
+  "version": "1.0.0",
+  "description": "A library of scripts for finding information about the ONSDigital github organisation",
+  "main": "index.js",
+  "repository": "git@github.com:ONSdigital/sdc-ia-scripts.git",
+  "author": "Matthew Elcock <mattiose@outlook.com>",
+  "license": "MIT",
+  "private": false,
+  "dependencies": {
+    "graphql-request": "^1.6.0",
+    "mkdirp": "^0.5.1",
+    "pkg": "^4.3.3"
+  },
+  "devDependencies": {
+    "eslint": "^5.1.0",
+    "eslint-config-eq-author": "^1.1.0",
+    "jest": "^23.4.1",
+    "lodash": "^4.17.10"
+  },
+  "scripts": {
+    "test": "yarn lint:js",
+    "lint:js": "eslint src --ext .js --max-warnings=0"
+  }
```

Comment:
Looks like the build script was not added in your latest commit. Otherwise looking good to me. Good work!